ci/azure-jobs-linux.yml

```diff
@@ -40,6 +41,9 @@ jobs:
   - bash: rm -rf libs/polyclipping
     displayName: Remove bundled polyclipping lib
     condition: and(succeeded(), contains(variables['UNBUNDLE'], 'polyclipping'))
+  - bash: rm -rf libs/hoedown
+    displayName: Remove bundled hoedown lib
+    condition: and(succeeded(), eq(variables['NO_HOEDOWN'], 'true'))
```

Comment:
This should ensure that building without hoedown actually works. (I already tested this on my development machine, but it's always better to check it in CI as well.)

Reply:
Sounds reasonable :+1: :slightly_smiling_face: